server/index.js

```diff
@@ -82,15 +87,18 @@ const startListening = () => {
 
 const syncDb = () => db.sync()
 
+async function bootApp () {
+  await sessionStore.sync()
+  await syncDb()
+  await createApp()
+  await startListening()
+}
```

Comment:
Is it worth doing `async`/`await` for everything? Maybe not. But I decided to be a completionist here.